ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs

```diff
@@ -88,6 +102,18 @@ internal class BiggestGridDetectionPcuStrategy : AbstractGridDetectionStrategy {
                     continue;
 
                 gridsList.Add(cubeGrid);
+                if (config.ExcludeProjectionPCU)
+                {
+                    List<MyProjectorBase> projs = new List<MyProjectorBase>();
+                    var gts = MyAPIGateway.TerminalActionsHelper.GetTerminalSystemForGrid(cubeGrid);
+                    gts.GetBlocksOfType(projs);
+                    foreach (var proj in projs)
```

Comment:
I am not a big fan of this naming. I mean "projector" and "projectors" and "gridTerminalSystem" is not that more complicated and makes it easer to understand the method.